packages/grafana-ui/src/components/DateTimePickers/DateTimePicker/DateTimePicker.tsx

```diff
@@ -86,6 +88,7 @@ export const DateTimePicker = ({ date, maxDate, label, onChange }: Props) => {
                   isFullscreen={true}
                   onClose={() => setOpen(false)}
                   maxDate={maxDate}
+                  minDate={minDate}
```

Comment:
I believe we should add `maxDate` and `minDate` to the `DateTimeCalendar` in line 104 as well, I don't see why `maxDate` wasn't added there. What do you think?

Reply:
Yeah, I think that makes sense, add both props there.